core/src/main/java/com/netease/arctic/table/UnkeyedTable.java

```diff
@@ -57,4 +58,6 @@ default boolean isUnkeyedTable() {
   default UnkeyedTable asUnkeyedTable() {
     return this;
   }
+
+  void dropPartitions(String partitions) throws TException, InterruptedException;
```

Comment:
I don't   think this feature should bring in a new method in the interface BaseUnkeyedTable